freud/environment.pyx

```diff
@@ -470,21 +471,20 @@ cdef class LocalDescriptors(Compute):
         return self.thisptr.getNPoints()
 
     @Compute._computed_property()
-    def num_neighbors(self):
+    def num_sphs(self):
         return self.thisptr.getNSphs()
 
     @property
     def l_max(self):
         return self.thisptr.getLMax()
 
     def __repr__(self):
-        return ("freud.environment.{cls}(num_neighbors={num_neigh}, "
+        return ("freud.environment.{cls}(num_neighbors={num_neighbors}, "
                 "lmax={lmax}, r_max={r_max}, "
-                "negative_m={negative_m})").format(cls=type(self).__name__,
-                                                   num_neigh=self.num_neigh,
-                                                   lmax=self.lmax,
-                                                   r_max=self.r_max,
-                                                   negative_m=self.negative_m)
+                "negative_m={negative_m})").format(
+                    cls=type(self).__name__, num_neighbors=self.num_neighbors,
+                    lmax=self.lmax, r_max=self.r_max,
```

Comment:
Does this even work? (Make sure we have a `repr` test for this.) The property above is `l_max`. We need to be consistent with `r_max` name conventions.